test/parallel/test-cluster-send-handle-twice.js

```diff
@@ -1,36 +1,36 @@
 'use strict';
 // Testing to send an handle twice to the parent process.
 
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
 
-var workers = {
+const workers = {
   toStart: 1
 };
 
 if (cluster.isMaster) {
-  for (var i = 0; i < workers.toStart; ++i) {
-    var worker = cluster.fork();
-    worker.on('exit', function(code, signal) {
-      assert.equal(code, 0, 'Worker exited with an error code');
-      assert(!signal, 'Worker exited by a signal');
-    });
+  for (let i = 0; i < workers.toStart; ++i) {
+    const worker = cluster.fork();
+    worker.on('exit', common.mustCall(function(code, signal) {
+      assert.strictEqual(code, 0, 'Worker exited with an error code');
+      assert.strictEqual(signal, null);
+    }));
   }
 } else {
-  var server = net.createServer(function(socket) {
+  const server = net.createServer(function(socket) {
     process.send('send-handle-1', socket);
     process.send('send-handle-2', socket);
   });
 
   server.listen(common.PORT, function() {
-    var client = net.connect({ host: 'localhost', port: common.PORT });
-    client.on('close', function() { cluster.worker.disconnect(); });
+    const client = net.connect({ host: 'localhost', port: common.PORT });
+    client.on('close', common.mustCall(function() { cluster.worker.disconnect(); }));
```

Comment:
This change isn't really necessary (although it doesn't hurt anything). Nothing is being asserted in the callback, so the test *should* fail if it's not called.